spec/support/file_structure.rb

```diff
@@ -7,7 +7,8 @@ class << self
     attr_accessor :root
 
     def have_git
-      FileUtils.mkdir_p(File.join(tmp, '.git', 'hooks'))
+      FileUtils.mkdir_p(File.join(tmp))
+      _, _, _ = Open3.capture3('git init', chdir: FileStructure.tmp)
```

Comment:
Is there any reason to reference `tmp` method with implicit and explicit receiver in adjacent lines? It is pretty confusing.
```suggestion
      FileUtils.mkdir_p(File.join(tmp))
      _, _, _ = Open3.capture3('git init', chdir: tmp)
```